Rakefile

```diff
@@ -116,6 +116,9 @@ end
 
 desc "Post build cleanup tasks"
 task :post do
+  # Put version file in place on VM
+  FileUtils.copy('./version.yaml','/etc/vm-version')
```

Comment:
Seems like the version.yaml file would be in /usr/src/p-t-b/ - do you think it necessary to specify the absolute path?
Also, why do we want to save the file, what does it do? (just curious).